angular/src/components/avatar.component.ts

```diff
@@ -1,138 +1,143 @@
-import {
-    Component,
-    Input,
-    OnChanges,
-    OnInit,
-} from '@angular/core';
-import { DomSanitizer } from '@angular/platform-browser';
+import { Component, Input, OnChanges, OnInit } from "@angular/core";
+import { DomSanitizer } from "@angular/platform-browser";
 
-import { CryptoFunctionService } from 'jslib-common/abstractions/cryptoFunction.service';
-import { StateService } from 'jslib-common/abstractions/state.service';
+import { CryptoFunctionService } from "jslib-common/abstractions/cryptoFunction.service";
+import { StateService } from "jslib-common/abstractions/state.service";
 
-import { Utils } from 'jslib-common/misc/utils';
+import { Utils } from "jslib-common/misc/utils";
 
 @Component({
-    selector: 'app-avatar',
-    template: '<img [src]="sanitizer.bypassSecurityTrustResourceUrl(src)" title="{{data}}" ' +
-        '[ngClass]="{\'rounded-circle\': circle}">',
+  selector: "app-avatar",
+  template:
+    '<img [src]="sanitizer.bypassSecurityTrustResourceUrl(src)" title="{{data}}" ' +
+    "[ngClass]=\"{'rounded-circle': circle}\">",
```

Comment:
No sure about moving the slash onto the beginning and removing the escaping of single-quotes for rounded-circle

Reply:
It's swapping the single quote with double.

`'[ngClass]="{\'rounded-circle\': circle}">',` => we need to escape the `'`.

`"[ngClass]=\"{'rounded-circle': circle}\">",` => now we need to escape `"`.